lumicks/pylake/population/mixture.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+from sklearn.mixture import GaussianMixture
+from scipy import stats
+
+
+def as_sorted(fcn):
+    """Decorator to return results sorted according to mapping array."""
+
+    def wrapper(self, *args, **kwargs):
+        result = fcn(self, *args, **kwargs)
+        return result[self._map]
+
+    return wrapper
+
+
+class GaussianMixtureModel:
+    def __init__(self, trace, K, init_method, n_init, tol, max_iter):
+        """A wrapper around scikit-learn's GMM
```

Comment:
Added a bit of explanation here. I think it sounds a bit awkward but let's see what you guys think. In my experience it's easier to compare different models (either different number of states or trained on different data) if the parameters are sorted somehow rather than being in random order (by means seems to make the most sense here). Also, this wrapper is specifically implemented for 1D data (thus the `squeeze()`, this way you don't have to worry about output shape)

Reply:
I think that you could add some of the info from this comment to the docstring. Sorting is an important aspect of this class and quite different from the original so it's good to point that out to users. Similarly for `squeeze()`.